core/src/main/resources/audio/article/index.scala.html

```diff
@@ -0,0 +1,46 @@
+@(atom: com.gu.contentatom.thrift.Atom
+, data: com.gu.contentatom.thrift.atom.audio.AudioAtom
+)(implicit conf: com.gu.contentatom.renderer.ArticleConfiguration)
+
+@fragments.atoms.html.audio(
+    atom.id,
+    className = "podcast",
+    label = "Today in Focus",   // TODO: get from data
```

Comment:
Actually `data.kicker` is the episode title, so we want to leave this hardcoded for now. We'll need to change the data model to add this in. Can do later

Reply:
Hmm, that's not good. The episode title must go [here](https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/contentatom.thrift#L109).

Reply:
Oh I see. Okay. That's pretty weird in the atom workshop UI because you add the episode title before the label. But can be done. Make Regis's changes and I'll change that test atom.

Reply:
I had forgotten about that terrible UI decision, sorry about that 😫